bindings/pyroot/pythonizations/python/ROOT/_pythonization/_function_utils.py

```diff
@@ -0,0 +1,224 @@
+import re
+from .._numbadeclare import _NumbaDeclareDecorator
+
+class FunctionSignature():
+    """
+    Class to determine a Function's Signature.
```

Comment:
We need some more docstrings here. Please elaborate a bit more on the goal of the class and what data members it has. You can use numpy style docstrings (see other examples of docstrings in this same file).